source/agora/test/Base.d

```diff
@@ -264,6 +265,12 @@ public final class TestNode (Net) : Node!Net, TestAPI
         return new MemMetadata();
     }
 
+    /// Return a transaction pool backed by an in-memory SQLite db
+    public override TransactionPool getPool (string _unused) @system
```

Comment:
There's a simple way to tell an argument is unused: don't give it a name ;)

Reply:
Ah I seem to remember ddoc complaining about it.. but maybe that was years ago. I'll fix.